agent/pkg/dagent/utils/import_container.go

```diff
@@ -39,38 +40,39 @@ func spawnInitContainer(
 		return err
 	}
 
-	builder := NewDockerBuilder(cli, cfg)
+	build := builder.NewDockerBuilder(cli)
```

Comment:
The original name is better.

Reason:
builder.Create() : is valid readable line, while in case of build.Create() we won't create builds here.

I would prefer a different name for the import and keep the original for the builder.